centreon/src/Core/Dashboard/Infrastructure/API/FindDashboard/FindDashboardPresenter.php

```diff
@@ -91,4 +94,17 @@ private function panelToArray(PanelResponseDto $panel): array
             'widget_settings' => (object) $panel->widgetSettings,
         ];
     }
+
+    /**
+     * @param RefreshResponseDto $refresh
+     *
+     * @return array
+     */
+    private function globalRefreshToArray(RefreshResponseDto $refresh): array
+    {
+        return [
+            'refresh_type' => RefreshTypeConverter::toString($refresh->refreshType),
+            'refresh_interval' => $refresh->refreshInterval,
```

Comment:
'type' and 'interval' would have been enought since your in the object 'refresh'

Reply:
Done